specification/web/resource-manager/Microsoft.Web/stable/2021-02-01/CommonDefinitions.json

```diff
@@ -151,6 +151,10 @@
           "description": "Dedicated Host Count",
           "type": "integer",
           "readOnly": true
+        },
+        "zoneRedundant": {
```

Comment:
should this have a default value like the ASP version?

Reply:
Since "null" is a meaningful value in the context of update requests to the hostingEnvironments resource, I think it makes sense to omit the default value here.